tests/performance/run_server_performance_test.py

```diff
@@ -0,0 +1,346 @@
+        before = datetime.datetime.now()
+        func(*args, **kwargs)
+        after = datetime.datetime.now()
+        return (after - before).total_seconds()
+
+    return func_wrapper
+
+
+def print_process_output(message, stdout, stderr):
+    global VERBOSE
+
+    if not VERBOSE:
+        return
+
+    print(message)
+    print('-' * 20 + 'stdout' + '-' * 20)
+    print(stdout)
+    print('-' * 20 + 'stderr' + '-' * 20)
+    print(stderr)
+    print('-' * (40 + len('stdout')))
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description='Performance tester for CodeChecker storage.',
+        epilog='This test simulates some user actions which are performed on '
+               'a CodeChecker server. The test instantiates the given number '
+               'of users. These users perform a run storage, some queries and '
+               'run deletion. The duration of all tasks is measured. These '
+               'durations are written to the output file at the end of the '
+               'test in CSV format. The tasks are performed for all report '
+               'directories by all users.')
+
+    parser.add_argument('input',
+                        type=str,
+                        metavar='file/folder',
+                        nargs='+',
+                        default='~/.codechecker/reports',
+                        help="The analysis result files and/or folders.")
+    parser.add_argument('--url',
+                        type=str,
+                        metavar='PRODUCT_URL',
+                        dest='product_url',
+                        default='localhost:8001/Default',
+                        required=True,
+                        help="The URL of the product to store the results "
+                             "for, in the format of host:port/ProductName.")
+    parser.add_argument('-o', '--output',
+                        type=str,
+                        required=True,
+                        help="Output file name for printing statistics.")
+    parser.add_argument('-u', '--users',
+                        type=int,
+                        default=1,
+                        help="Number of users")
+    parser.add_argument('-t', '--time',
+                        type=int,
+                        default=-1,
+                        help="Timout in seconds. The script stops when the "
+                             "timeout expires. If a negative number is given "
+                             "then the script runs until it's interrupted.")
+    parser.add_argument('-b', '--beta',
+                        type=int,
+                        default=10,
+                        help="In the test users are waiting a random amount "
+                             "of seconds. The random numbers have exponential "
+                             "distribution of the beta parameter can be "
+                             "provided here.")
+    parser.add_argument('-v', '--verbose',
```

Comment:
I started to run this script without a running server. The output of this script was the following:
```bash
User 1 is sleeping 0.391030774259 seconds
Storage of performance_test_1 is started (~/.codechecker/tinyxml/)
Storage of performance_test_1 is done
```
This tells me that the storage was successful.

When I run this test script with the verbose flag the output says the storage was unsuccessful.
```bash
User 1 is sleeping 5.28858773684 seconds
Storage of performance_test_1 is started (/home/ecsomar/.codechecker/tinyxml-resolved/)
Output of storage
--------------------stdout--------------------
[09:39] - Storing analysis results for run 'performance_test_1'
Connection refused
[Errno 111] Connection refused

--------------------stderr--------------------

----------------------------------------------
Storage of performance_test_1 is done
```
Wouldn't it be better to write a better message to the user that the storage was successful or unsuccessful?